public/v2/apps/eclipse-mosquitto.json

```diff
@@ -0,0 +1,60 @@
+{
+    
+"captainVersion": "2",
+"documentation": "Taken from https://github.com/eclipse/mosquitto. Mosquitto is an open source implementation of a server for version 5.0, 3.1.1, and 3.1 of the MQTT protocol. It also includes a C and C++ client library, and the mosquitto_pub and mosquitto_sub utilities for publishing and subscribing.",
+"displayName": "Eclipse Mosquitto - A MQTT Broker",
+"description": "Eclipse Mosquitto is an open source message broker which implements MQTT version 5, 3.1.1 and 3.1",
+
+
+
+"dockerCompose": {
+    "version": "3.3",
+    "services": {
+        "$$cap_appname": {
+
+  
+            "image": "eclipse-mosquitto:$$cap_eclipse_version",
+            "volumes": [
+                "$$cap_appname-eclipse-config:/mosquitto/config/",
+                "$$cap_appname-eclipse-data:/mosquitto/data/",
+                "$$cap_appname-eclipse-log:/mosquitto/log/"
+            ],
+            "notExposeAsWebApp": "true",
+            "ports": [
+                    "$$cap_mqtt_port:1883",
+                    "$$cap_ws_port:9001"],
+        "restart": "always"
+  }
+}
+},
+
+"instructions": {
+    "start": "Eclipse Mosquitto is an open source message broker which implements MQTT version 5, 3.1.1 and 3.1. Enter Parameters below",
+    "end": "Eclipes-Mosquitto is deployed and available as srv-captain--$$cap_appname:$$cap_mqtt_port to other apps and srv-captain--$$cap_appname:$$cap_ws_port as a websocket."
```

Comment:
Are you sure you tested these and it worked? `srv-captain--$$cap_appname:$$cap_mqtt_port`  and `srv-captain--$$cap_appname:$$cap_ws_port` are incorrect.
As mentioned in the other comments these are the host ports. When you're using `srv-captain--*****` you are binding to the container ports. There for these two need to be changed to
```
srv-captain--$$cap_appname:1883 [mqtt port]
srv-captain--$$cap_appname:9001 [websocket]
```

i.e. when you are binding locally to these services, the ports are always 1883 and 9001. See other apps, for example [MySQL](https://github.com/caprover/one-click-apps/blob/master/public/v2/apps/mysql.json#L22) for more details.